IBPSA/Fluid/HeatPumps/ModularReversible/LargeScaleWaterToWater.mo

```diff
@@ -0,0 +1,106 @@
+    final mCon_flow_nominal=autCalMasCon_flow,
+    final mEva_flow_nominal=autCalMasEva_flow,
+    final tauCon=autCalVCon*rhoCon/autCalMasCon_flow,
+    final tauEva=autCalVEva*rhoEva/autCalMasEva_flow);
+
+  extends IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.LargeScaleWaterToWaterDeclarations(
+    final autCalMasCon_flow=max(4E-5*QUse_flow_nominal - 0.6162, autCalMMin_flow),
+    final autCalMasEva_flow=max(4E-5*QUse_flow_nominal - 0.3177, autCalMMin_flow),
+    final autCalVCon=max(1E-7*QUse_flow_nominal - 94E-4, autCalVMin),
+    final autCalVEva=max(1E-7*QUse_flow_nominal - 75E-4, autCalVMin));
+
+  replaceable parameter
+    IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.GenericHeatPump datTab
+    "Data table of heat pump" annotation (choicesAllMatching=true,
+    Placement(transformation(extent={{42,12},{58,28}})));
+
+  replaceable parameter
+    IBPSA.Fluid.HeatPumps.ModularReversible.Controls.Safety.Data.Wuellhorst2021 safCtrParEurNor
+      constrainedby IBPSA.Fluid.HeatPumps.ModularReversible.Controls.Safety.Data.Generic(
+    final tabUppHea=datTab.tabUppBou,
+    final tabLowCoo=datTab.tabUppBou,
+    final use_TUseSidOut=datTab.use_TConOutForOpeEnv,
+    final use_TAmbSidOut=datTab.use_TEvaOutForOpeEnv) "Safety control parameters"
+    annotation (Dialog(enable=use_intSafCtr, group="Safety control"),
+      choicesAllMatching=true,
+      Placement(transformation(extent={{72,12},{88,28}})));
+    // Lower boundary has no influence as use_rev=false
+  annotation (Documentation(info="<html>
+<p>
+  Model using parameters for a large scale water-to-water heat pump,
+  using the ModularReversible model approach.
+</p>
+<p>
+  Contrary to the standard sizing approach for the
+  <a href=\"IBPSA.Fluid.HeatPumps.ModularReversible.ModularReversible\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.ModularReversible</a> models,
+  the parameters are based on an automatic estimation as described in
+  <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.LargeScaleWaterToWaterDeclarations\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses.LargeScaleWaterToWaterDeclarations</a>.
+</p>
+<p>
+  For more information on the approach, please read the
+  <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.UsersGuide\">
+  UsersGuide</a>.
+</p>
+<p>
+  Please read the documentation of the model for heating here:
+  <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2D</a>.
+</p>
+<p>
+  Currently the only data sheets for heat pumps that large is the record
+  <a href=\"modelica://IBPSA.Fluid.HeatPumps.ModularReversible.Data.TableData2D.EN14511.WAMAK_WaterToWater_150kW\">
+  IBPSA.Fluid.HeatPumps.ModularReversible.RefrigerantCycle.TableData2DData.EN14511.WAMAK_WaterToWater_150kW</a>,
+  hence, the default value.
```

Comment:
I don't understand this comment as there is no default value. (There should be no default value because otherwise, once we add more data records, we should remove the default value which becomes a backward incompatible change. In comparison, for the fans, we also don't set a default value).

Reply:
This was a relict, thanks for noticing.